test/truffle/cexts/backtraces/expected.txt

```diff
@@ -67,7 +67,7 @@ Caused by:
 	from /lib/truffle/truffle/cext.rb:n:in `execute_with_mutex'
 	from /lib/truffle/truffle/cext_ruby.rb:n:in `native_callback'
 	from /bin/backtraces:92:in `<main>'
-Translated to internal error (RubyTruffleError)
+Translated to internal error (RuntimeError)
```

Comment:
This sounds confusing now. Internal errors have the same class as Ruby user-level exceptions by `raise "foo"` :disappointed: